web_responsive/static/src/css/variables.scss

```diff
@@ -0,0 +1,2 @@
+$o-brand-odoo: $o-community-color;
+$o-brand-primary: $o-community-color;
```

Comment:
You don't need to declare these variables, as [they are already found in Odoo core](https://github.com/odoo/odoo/blob/528e10572195bda2102648b935d6d8647334fd4b/addons/web/static/src/scss/primary_variables.scss#L16-L17).